lib/std/cstr.zig

```diff
@@ -28,7 +28,7 @@ test "cstr fns" {
 
 fn testCStrFnsImpl() void {
     testing.expect(cmp("aoeu", "aoez") == -1);
-    testing.expect(mem.len(u8, "123456789") == 9);
+    testing.expect(mem.len(u8, "123456789".*) == 9);
```

Comment:
The way the test was before is testing what it intends to test - calculating the length of a null terminated pointer. e.g. the zig equivalent of `strlen`.

Reply:
Should it perhaps be `.ptr` instead?

Reply:
The type of a string literal is `*const [9:0]u8`; IMO asking for the length of a pointer seems weird, and we need to `.*` to make sense...